helm/prometheus-rules/templates/kaas/turtles/alerting-rules/node.workload-cluster.rules.yml

```
@@ -165,3 +165,14 @@ spec:
         severity: page
         team: se
         topic: infrastructure
+    - alert: WorkloadClusterNodeUnexpectedTaintNodeCAPIUninitialized
+      annotations:
+        description: '{{`Node {{ $labels.node }} in cluster {{ $labels.installation }}/{{ $labels.cluster_id }} has CAPI taint node.cluster.x-k8s.io/uninitialized for too long`}}'
+        opsrecipe: unexpected-taint-capi/
+      expr: kube_node_spec_taint{key="node.cluster.x-k8s.io/uninitialized"} > 0
+      for: 15m
+      labels:
+        area: kaas
+        severity: page
+        team: phoenix
```

```suggestion
        team: tenet
```

I'm happy for y'all to have this alert but I suspect its really for us, right?